blocks/cards/cards.css

```diff
@@ -1,27 +1,64 @@
-.cards > ul {
+ .cards > ul {
   list-style: none;
-  margin: 0;
-  padding: 0;
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(257px, 1fr));
-  grid-gap: 24px;
+  grid-template-columns: 1fr; /* Default: single column for mobile */
+  grid-gap: 16px; /* Space between cards */
+} 
+
+.cards > ul > li{
+  position: relative;
+  margin:20px;
+}
+
+.cards > ul > li >p{
+  margin:0;
 }
 
-.cards > ul > li {
-  border: 1px solid #dadada;
-  background-color: var(--background-color);
+.cards-card-para{
+display: block;
+} 
+
+.cards-card-para.hidden {
+  display: none;
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.cards-card-body { /* Match your image bg color */
+text-align: center;
+background: #484646;
+padding: 80px 40px 40px;
+  color: #fff;
```

Comment:
the alignment of properties within rule should be same across the file. currently there isnt any space (indentation should be even across the file